script/prettier.ts

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env ts-node
+
+import * as Path from 'path'
+import chalk from 'chalk'
+import { spawnSync } from 'child_process'
+
+const shouldFix = process.argv.indexOf('--fix') > -1
+
+const root = Path.dirname(__dirname)
+
+const prettier = process.platform === 'win32' ? 'prettier.cmd' : 'prettier'
+const prettierPath = Path.join(root, 'node_modules', '.bin', prettier)
+
+const args = ['**/*.{md,scss}']
+
+if (shouldFix) {
+  args.push('--write')
+} else {
+  args.push('--list-different')
```

Comment:
IMO we should always pass this argument — when used with `--write`, it only prints out the files it changed, instead of printing out every file’s path.

Reply:
> it only prints out the files it changed, instead of printing out every file’s path.

By passing in both values, you'll get the fixes applied _and_ it'll return a non-zero exit code. 

We could do something like this:

```shellsession
~/src/desktop/ yarn lint:prettier --fix
yarn run v1.3.2
$ ts-node script/prettier.ts --fix
These files were formatted by prettier:

app/styles/ui/_repository-list.scss
docs/contributing/linux-testing-and-you.md
docs/technical/syntax-highlighting.md
```

But then we now need more exit code handling. We're not doing anything like this in `script/eslint.ts`, so I'm inclined to follow that for now unless things get really confusing for contributors...

Reply:
It shouldn’t be that bad, just put `&& !shouldFix` after `result.status > 0` on line 26.